optimizely/client.py

```diff
@@ -12,10 +12,11 @@
 class Client(object):
   ALLOWED_REQUESTS = ['get', 'post', 'put', 'delete']
 
-  def __init__(self, api_key, api_base=BASE_URL):
+  def __init__(self, api_key, api_base=BASE_URL, token_type):
```

Comment:
positional args should be before any keyword args. i don't think this code will even run, will it?